docs/layers/base-layer.md

```diff
@@ -107,33 +96,26 @@ Layers can be interacted with using these properties.
 
 - Default: `false`
 
-Whether layer responds to mouse events.
+Whether layer responds to mouse pointer picking events.
 
 ##### `onHover` (Function, optional)
 
 This callback will be called when the mouse hovers over the deck.gl
-viewport. The callback will be called for every layer that has
-both `visible` and `picking` properties set. When hovering over a
-feature drawn by the layer will be detected and mapped to an index.
-
-The function will be called with a single parameter `info`, which is an
-object that contains a variety of fields describing the mouse or touch
+viewport. The callback will be called for the layer that has
+been picked and the feature that user's mouse is hovered on will be
```

Comment:
Maybe use `object` or `data item`/`data object` here instead of feature - feature is very GeoJson specific.